08.ls_object/short_formatter.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class ShortFormatter
+  COLUMN = 3
+
+  def initialize(files)
+    @files = files
+  end
+
+  def output
+    max_length = @files.max_by(&:length).length
+
+    split_files_by_column.each do |files|
+      files.each do |file|
+        print format("%-#{max_length + 1}s", file)
+      end
+      puts
+    end
```

Comment:
参考）こういうふうに書いた方がよりシンプルになりそうです。

```suggestion
    length = @files.max_by(&:length).length + 1

    split_files_by_column.each do |files|
      files.each do |file|
        print file.ljust(length)
      end
      puts
    end
```

ただし、この場合は `divided_directories.last << ''` とする必要があります。